src/esp/assets/Attributes.h

```diff
@@ -182,6 +182,25 @@ class PhysicsManagerAttributes : public esp::core::Configuration {
   void setMaxSubsteps(int maxSubsteps) { setInt("maxSubsteps", maxSubsteps); }
   int getMaxSubsteps() const { return getInt("maxSubsteps"); }
 
+  void setGravity(const Magnum::Vector3& gravity) {
+    setVec3("gravity", gravity);
+  }
+  Magnum::Vector3 getGravity() const { return getVec3("gravity"); }
+
+  void setFrictionCoefficient(double frictionCoefficient) {
+    setDouble("frictionCoefficient", frictionCoefficient);
+  }
+  double getFrictionCoefficient() const {
+    return getDouble("frictionCoefficient");
+  }
+
+  void setRestitutionCoefficient(double restitutionCoefficient) {
+    setDouble("restitutionCoefficient", restitutionCoefficient);
+  }
+  double getRestitutionCoefficient() const {
+    return getDouble("restitutionCoefficient");
+  }
+
```

Comment:
This is information is already present in the `PhysicsSceneAttributes`. This distinction is not functional currently, but philosophically a `PhysicsManager` steps the world, while a scene has gravity, friction, and restitution.

Ideally, a single PhysicsManager could hold multiple scenes with different worlds (TODO).

Reply:
Should i remove these then?  I only added them because the equivalent fields were being set in PhysicsManagerAttributes via string keys, but i much prefer the scenario where a scene completely manages these world-related quantities and the PhysicsManager only steps the world.

Reply:
On the other hand, having these set in PhysicsManagerAttributes makes them able to be easily broadcast to the scenes, since the physics manager attributes are read first.